Azure/cloud-init.txt

```diff
@@ -73,17 +73,9 @@ apt-get install -y thin-provisioning-tools
 
 # Configure the clear containers runtime using devicemapper as
 # storage driver
-mkdir -p /etc/systemd/system/docker.service.d/
-cat <<EOF | tee /etc/systemd/system/docker.service.d/clr-containers.conf
-[Service]
-ExecStart=
-ExecStart=/usr/bin/dockerd -D --add-runtime cc-runtime=/usr/bin/cc-runtime --default-runtime=cc-runtime
-EOF
-
-# Add devicemapper configuration
 mkdir -p /etc/docker
-
 mkdir -p /etc/lvm/profile
+
 cat <<EOF | tee /etc/docker/daemon.json
```

Comment:
Not strictly part of this PR, but this should probably be `tee -a` to avoid stamping on whatever was there before.

Reply:
@jodh-intel I think we need to write a little utility to insert our runtime into the json file. This is also eliminate the one manual step we have today to install CC. We should build it into cc-runtime itself to help install itself.

Reply:
#763 raised to discuss this idea.